std/src/std/array.inko

```diff
@@ -449,7 +449,18 @@ type builtin Array[T] {
   # numbers.size # => 5
   # ```
   fn pub mut append(other: Array[T]) {
-    for v in other { push(v) }
+    if other.size == 0 { return }
+
+    let missing_capacity = (size + other.size) - capacity
+    if missing_capacity > 0 { reserve(missing_capacity) }
+
+    let from_addr = other.address_of(0)
+    let to_addr = address_of(size)
+    alloc.copy(from_addr, to_addr, other.size)
+    @size += other.size
+
+    # we just moved all this stuff, make sure `other` doesn't drop it
+    other.size = 0
```

Comment:
@Keithcat1 Thanks! I have the following small remarks:

- Could you run `inko fmt` and include any changes this makes? Looking at how the code is formatted, I think you forgot :)
- I added `Array.reserve_exact` in 66e139218023ff9a76c362b514ad6026091392e7 to make implementing this method a little easier.
- Instead of using `address_of(0)` for the source, you can use `other.pointer` 
- Basically you should be able to copy most of what `Bytes.append` does